### PR TITLE
Update onboard-configure with server versions

### PR DESCRIPTION
**Pull request by wahidsaleemi**
Added the Windows Server versions that are now supported. The list was taken from the linked article.

### CODE REVIEW

**Comment by illfated** [comment]
I agree with the addition of Server 2018, 2019, and SAC 1803, but I thought 2008 R2 reached End Of Support about a year ago. Are there any exclusions to the EOS date?

**Review by adirdidi — APPROVED** [review]

**Comment by adirdidi** [comment]
It's supported by MDE.

**Review by illfated — APPROVED** [review]
Very well, thank you for the explanation. Looks good and valid to me.

File: windows/security/threat-protection/microsoft-defender-atp/onboard-configure.md [hunk 1]
```diff
@@ -60,14 +60,10 @@ Topic | Description
 :---|:---
 [Onboard previous versions of Windows](onboard-downlevel.md)| Onboard Windows 7 and Windows 8.1 devices to Defender for Endpoint. 
 [Onboard Windows 10 devices](configure-endpoints.md) | You'll need to onboard devices for it to report to the Defender for Endpoint service. Learn about the tools and methods you can use to configure devices in your enterprise.
-[Onboard servers](configure-server-endpoints.md) |  Onboard Windows Server 2012 R2 and Windows Server 2016 to Defender for Endpoint 
+[Onboard servers](configure-server-endpoints.md) |  Onboard Windows Server 2008 R2 SP1, Windows Server 2012 R2, Windows Server 2016, Windows Server (SAC) version 1803 and later, Windows Server 2019 and later, and Windows Server 2019 core edition to Defender for Endpoint
```
Comment (illfated) [inline windows/security/threat-protection/microsoft-defender-atp/onboard-configure.md:63]: Cosmetic/minor: may we have a period dot at the end of the sentence?
```suggestion
[Onboard servers](configure-server-endpoints.md) |  Onboard Windows Server 2008 R2 SP1, Windows Server 2012 R2, Windows Server 2016, Windows Server (SAC) version 1803 and later, Windows Server 2019 and later, and Windows Server 2019 core edition to Defender for Endpoint.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - only @illfated's suggestion.

**Comment by e0i** [comment]
Hello @wahidsaleemi Would you be able to merge the copy editing suggestions mentioned above? The PR is good to go once those changes have been applied. Thank you for your contributions.

**Review by illfated — APPROVED** [review]
Looks good to me.

**Comment by illfated** [comment]
#sign-off

**Review by JohanFreelancer9 — APPROVED** [review]

**Pull request merged**